src/Sulu/Component/Rest/ListBuilder/ListRestHelper.php

```diff
@@ -22,6 +22,13 @@
  */
 class ListRestHelper implements ListRestHelperInterface
 {
+    const PARAMETER_FIELDS = 'fields';
+    const PARAMETER_PAGE = 'page';
+    const PARAMETER_SEARCH = 'search';
+    const PARAMETER_SEARCH_FIELDS = 'searchFields';
+    const PARAMETER_SORT_BY = 'sortBy';
+    const PARAMETER_SORT_ORDER = 'sortOrder';
```

Comment:
@danrot when I think about implement another LIstRestHelper we could add them to the __construct as parameters? so its possible to create an own listresthelper service with custom parameter names for an website application.  e.g.:

```
__construct(
    RequestStack $requestStack,
    $pageParameter = 'page',
    $limitParameter = 'limit',
    $sortParameter = 'sortBy',
    $sortOrderParameter = 'sortOrder',
    $searchParameter = 'search',
    $searchFieldsParameter = 'searchFields',
    $fields = 'fields'
) {
   // ...
}
```